libmamba/include/mamba/core/link.hpp

```diff
@@ -23,6 +23,16 @@
 namespace mamba
 {
     std::string replace_long_shebang(const std::string& shebang);
+    std::string python_shebang(const std::string& python_exe);
```

Comment:
These functions look like implementation functions (there are not called anywhere else in the library and I'm not sure they will), they could be moved to the cpp file (and forward declared in the test). We can also do a pass on this during the refactoring of Mamba APIs.

Reply:
that was like that before, but Windows complained in the test that it couldn't find the symbol :/